cfgov/v1/jinja2/v1/includes/molecules/contact-phone.html

```diff
@@ -7,7 +7,6 @@
    Description:
 
    Create a Contact-Phone molecule.
-   See [GHE]/flapjack/Modules-V1/wiki/Contact:-Phone
```

Comment:
Any interest in globally stripping out these /flapjack/ URL references (in another PR)?

Reply:
Done in #8396.